src/drivers/wdm/queries/PendingStatusError/PendingStatusError.ql

```diff
@@ -29,8 +29,27 @@ predicate isIOCompletionRoutine(Function f) {
 predicate returnsStatusPending(FunctionCall call) {
   exists(ReturnStmt rs |
     //259 is the integer representaion for STATUS_PENDING
-    rs.getExpr().(Literal).getValue().toInt() = 259 and
-    call.getASuccessor*() = rs
+    (

```

Comment:
Due to GitHub limitations I can't add this to the correct line, but please add a reference in the query description to the CA query ID this is a port of ([C28143](https://docs.microsoft.com/en-us/windows-hardware/drivers/devtest/28143-iomarkirppending-must-return-statuspending))

Reply:
reference added